### ISSUE TITLE
v2.2.0 broken in node 4

### ISSUE THREAD

**Issue by kellyselden** [chbrown/rfc6902#24]
https://travis-ci.org/kellyselden/rfc6902-ordered/builds/285767070#L525

```
/home/travis/build/kellyselden/rfc6902-ordered/node_modules/rfc6902/pointer.js:41
    constructor(tokens = ['']) {
                       ^
SyntaxError: Unexpected token =
```

Your CI may be testing something other than your compiled dist files.

**Comment by chbrown** [comment]
Actually the CI misrepresentation was that my tests were pre-compiling with babel, which isn't the typical environment :(

Should be fixed in 2.2.1: https://travis-ci.org/chbrown/rfc6902/builds/285901108

Please re-open if not.

**Comment by kellyselden** [comment]
Working now. Thanks for the quick turnaround!
